### PR TITLE
Create persistent volume to store log files

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: deployment/roles/deploy/templates/log-files-pvc.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# SPDX-License-Identifier: MIT
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: log-files-pvc
+spec:
+  accessModes:
+    - ReadWriteMany
```
Comment (jpopelka) [inline deployment/roles/deploy/templates/log-files-pvc.yml.j2:11]: The question is whether `RWX` mode is actually supported by the cluster we're about to deploy to.
Can you try first?

**Review by jpopelka — APPROVED** [review]

File: deployment/roles/deploy/templates/log-files-pvc.yml.j2 (hunk 1 above)
Comment (lbarcziova) [reply deployment/roles/deploy/templates/log-files-pvc.yml.j2:11]: yes, I will try

Comment (lbarcziova) [reply deployment/roles/deploy/templates/log-files-pvc.yml.j2:11]: I checked in the web ui and there was an option RWX available when I opened PVC creation options:
```
Access Mode
Single User (RWO) Shared Access (RWX) Read Only (ROX)

```

Comment (jpopelka) [reply deployment/roles/deploy/templates/log-files-pvc.yml.j2:11]: Ok, it should work then.
But I'd still suggest to try to create a PVC from your template before you merge this - just to be sure it doesn't spit out any errors later.

Comment (lbarcziova) [reply deployment/roles/deploy/templates/log-files-pvc.yml.j2:11]: I understand, I deployed just the PVC and it went well.

**Pull request merged**